test/surgex/repo_helpers/repo_helpers_test.exs

```diff
@@ -78,4 +78,56 @@ defmodule Surgex.RepoHelpersTest do
     final_opts = RepoHelpers.set_ssl([], "TEST_DB_SSL")
     refute Keyword.has_key?(final_opts, :ssl)
   end
+
+  test "set_application_name no env var, no existing params" do
+    System.delete_env("APP_NAME")
+    opts = RepoHelpers.set_application_name([])
+    refute Keyword.has_key?(opts, :parameters)
+  end
+
+  test "set_application_name no env var, existing params" do
+    System.delete_env("APP_NAME")
+
+    opts =
+      RepoHelpers.set_application_name(
+        parameters: [test_key: "test_value", application_name: "surgex"]
+      )
+
+    assert opts[:parameters][:test_key] == "test_value"
+    assert opts[:parameters][:application_name] == "surgex"
+  end
+
+  test "set_application_name env var present, no existing params" do
+    System.put_env("APP_NAME", "app-name-from-env")
+    opts = RepoHelpers.set_application_name([])
+    assert opts[:parameters][:application_name] == "app-name-from-env"
+    System.delete_env("APP_NAME")
```

Comment:
Shouldn't this be in the `after` clause?

Reply:
Is there an `after` clause in ExUnit? 
I've found `on_exit`, and I've pushed the commit that uses it.

Reply:
It turns out `after` is a elixir feature: https://elixir-lang.org/getting-started/try-catch-and-rescue.html#after